js/components/tabs.js

```diff
@@ -27,6 +27,25 @@ class Tabs extends ImmutableComponent {
     this.onNextPage = this.onNextPage.bind(this)
     this.onNewTabLongPress = this.onNewTabLongPress.bind(this)
     this.wasNewTabClicked = this.wasNewTabClicked.bind(this)
+    this.onMouseLeave = this.onMouseLeave.bind(this)
+    this.onFrameClose = this.onFrameClose.bind(this)
+
+    this.state = {
+      fixTabWidth: null
+    }
+  }
+
+  onMouseLeave () {
+    this.setState({
+      fixTabWidth: null
+    })
+    this.forceUpdate()
```

Comment:
I have to use forceUpdate here because we inherit from `ImmutableComponent` and not `ReactComponent`. `ImmutableComponent` checks for props only to make the render update work.
forcedUpdate has been used in other places also (possibly because of the above reason).